src/PluginsSystem/Tools/Microsoft.Performance.Toolkit.Plugins.Cli/IPack.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.Toolkit.Plugins.Cli.Options;
+
+namespace Microsoft.Performance.Toolkit.Plugins.Cli
+{
+    internal interface IPack
```

Comment:
Instead of having a bunch of interfaces, you can just create a single generic one:

```cs
internal interface IPluginToolkitAction<T>
    where T : PackageGenCommonOptions
{
    int Run(T options);
}
```